backend/notifications/NotificationService.cs

```diff
@@ -171,7 +172,18 @@ public async Task CancelAsync(IEnumerable<Entity.NotificationQueue> notification
 
             foreach (var notification in notifications.Where(n => n.ChesMessageId.HasValue))
             {
-                await CancelAsync(notification);
+                try
+                {
+                    await CancelAsync(notification);
+                }
+                catch (HttpClientRequestException ex)
+                {
+                    // CHES returns a 404 when a message doesn't exist.  This isn't a good practice, but now I have to assume 404's are successful and I'm sending invalid message Ids.
+                    if (ex.StatusCode != HttpStatusCode.NotFound) throw;
```

Comment:
A 404 regrettably is just a missing message (caused by the CHES migration to OCP4).  We'll ignore these for now and continue cancelling all other messages.